### ISSUE TITLE
AWS018 failures

### ISSUE THREAD

**Issue by sercanacar** [aquasecurity/tfsec#869]
**Describe the bug**
TFsec is failing with AWS018

```
Resource 'aws_security_group_rule.untrust-rules' should include a non-empty description for auditing purposes.
```

Terraform
```
resource "aws_security_group_rule" "trust-rules-dev" {
  count = ...
  description       = var.trust-sg-rules[count.index]["description"]
...

variable "trust-sg-rules" {
  description = "A list of maps that creates a number of security group rules for the trust interface"
  type        = list(map(string))
...
```
Version 0.45.0

What am I doing wrong?

**Comment by owenrumney** [comment]
Hey @sercanacar - have you got a default value for `trust-sg-rules` or are you passing in a tfvars file? If neither, then tfsec won't find a value for the description

**Comment by owenrumney** [comment]
I'll take a look and try to reproduce

**Comment by owenrumney** [comment]
@sercanacar  - we've fixed this in `v0.48.0` which is currently building. It seems we weren't actively supporting `count` so that's now been added in #871

**Comment by sercancontino** [comment]
That's awesome, thank you for the quick turnaround
